docs/vars.md

```diff
@@ -136,14 +136,12 @@ Stack](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/dns-stack.m
   [kubelet-rubber-stamp](https://github.com/kontena/kubelet-rubber-stamp).
 * *node_labels* - Labels applied to nodes via kubelet --node-labels parameter.
   For example, labels can be set in the inventory as variables or more widely in group_vars.
-  *node_labels* can be defined either as a dict or a comma-separated labels string:
+  *node_labels* can only be defined as a dict: 
```

Comment:
The markdownlint detects the following issue:
```
$ markdownlint $(find . -name '*.md' | grep -vF './.git') --ignore docs/_sidebar.md --ignore contrib/dind/README.md
./docs/vars.md:139:47 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
```
Please remove a space at the end of line.

Reply:
```suggestion
  *node_labels* can only be defined as a dict:
```